src/App.tsx

```diff
@@ -1,12 +1,62 @@
-import React from 'react';
```

Comment:
Removing this will cause the app to fail after compiling. I think this should also trigger an error during local development. I don't remember well but I think it's mandatory to have react imported for every file that implements a function component.

Reply:
Oh, okay! Sorry but I've heard that in the new version of React is not necessary import react

Reply:
Oh, my fault then! #TheMoreYouKnow